### DISCUSSION TITLE
osgearth_conv is not working as expected

### DISCUSSION THREAD

**Discussion by christ0phd** [pelicanmapping/osgearth#2301] [Support Q&A]
Hey all,
I use osgearth_conv to produce a TMS dataset from a geotiff. Starting the tool with the minimum set of options documented [here](http://docs.osgearth.org/en/latest/data.html) the maximum level is 5. As the relation of height ot width of the geotiff is close to 2 to 1 the generated TMS tiles look very compressed. Using those tiles in our SW the image looks quite blurry :-( So I tried to fine tune the osgearth_conv call.

- Increasing the maximum level by passing --max-level 6 is reported by the textual output in the shell but the SW stops after the 5th level and produces the same amount of output as with level 5.
- Setting minimum and maximum level results also with the same result as not specifying the maximum level.
- Tried to set numTilesWideAtLod0 and numTilesHighAtLod0 somehow with the cli did not work at all
- Created a small earth file containing a profile with set numTilesWideAtLod0 and numTilesHighAtLod0 influenced the amount of estimated tiles. But the result was still the same as without these setting.

So my question is, how can I avoid the blurry image? Is it the right way to finetune the osgearth_conv call? What would be the correct option to do so? Or is there a different working way?

Christoph

**Comment by gwaldron** [comment]
If you have the `gdalinfo` tool, can you please post the output of `gdalinfo file.tif`?
Otherwise, can you please set your environment variable `OSGEARTH_NOTIFY_LEVEL=INFO` and post the output of osgearth_conv?

**Reply by christ0phd** [reply]
The gdalinfo output is the following:
`Driver: GTiff/GeoTIFF`
`Files: input.tif`
`Size is 7896, 12690`
`Coordinate System is:`
`PROJCRS["WGS 84 / UTM zone 32N",`
`    BASEGEOGCRS["WGS 84",`
`        DATUM["World Geodetic System 1984",`
`            ELLIPSOID["WGS 84",6378137,298.257223563,`
`                LENGTHUNIT["metre",1]]],`
`        PRIMEM["Greenwich",0,`
`            ANGLEUNIT["degree",0.0174532925199433]],`
`        ID["EPSG",4326]],`
`    CONVERSION["UTM zone 32N",`
`        METHOD["Transverse Mercator",`
`            ID["EPSG",9807]],`
`        PARAMETER["Latitude of natural origin",0,`
`            ANGLEUNIT["degree",0.0174532925199433],`
`            ID["EPSG",8801]],`
`        PARAMETER["Longitude of natural origin",9,`
`            ANGLEUNIT["degree",0.0174532925199433],`
`            ID["EPSG",8802]],`
`        PARAMETER["Scale factor at natural origin",0.9996,`
`            SCALEUNIT["unity",1],`
`            ID["EPSG",8805]],`
`        PARAMETER["False easting",500000,`
`            LENGTHUNIT["metre",1],`
`            ID["EPSG",8806]],`
`        PARAMETER["False northing",0,`
`            LENGTHUNIT["metre",1],`
`            ID["EPSG",8807]]],`
`    CS[Cartesian,2],`
`        AXIS["(E)",east,`
`            ORDER[1],`
`            LENGTHUNIT["metre",1]],`
`        AXIS["(N)",north,`
`            ORDER[2],`
`            LENGTHUNIT["metre",1]],`
`    USAGE[`
`        SCOPE["Engineering survey, topographic mapping."],`
`        AREA["Between 6°E and 12°E, northern hemisphere between equator and 84°N, onshore and offshore. Algeria. Austria. Cameroon. Denmark. Equatorial Guinea. France. Gabon. Germany. Italy. Libya. Liechtenstein. Monaco. Netherlands. Niger. Nigeria. Norway. Sao Tome and Principe. Svalbard. Sweden. Switzerland. Tunisia. Vatican City State."],`
`        BBOX[0,6,84,12]],`
`    ID["EPSG",32632]]`
`Data axis to CRS axis mapping: 1,2`
`Origin = (xMin,yMax)`
`Pixel Size = (7.092198581560283,-7.092198581560283)`
`Metadata:`
`  AREA_OR_POINT=Area`
`  TIFFTAG_RESOLUTIONUNIT=3 (pixels/cm)`
`  TIFFTAG_XRESOLUTION=70.5`
`  TIFFTAG_YRESOLUTION=70.5`
`Image Structure Metadata:`
`  INTERLEAVE=PIXEL`
`Corner Coordinates:`
`Upper Left  (  xMin, yMax) (  topLeftE, topLeftN)`
`Lower Left  (  xMin, yMin) (  bottomLeftE, bottomLeftN)`
`Upper Right (  xMax, yMax) (  topRightE, topRightN)`
`Lower Right (  xMax, yMin) (  bottomRightE, bottomRightN)`
`Center      (  xCenter, yCenter) (  centerE, centerN)`
`Band 1 Block=7896x1 Type=Byte, ColorInterp=Red`
`Band 2 Block=7896x1 Type=Byte, ColorInterp=Green`
`Band 3 Block=7896x1 Type=Byte, ColorInterp=Blue`

Do you need any further information?

Christoph
